packages/cli/src/bin/dev-portal/Header.tsx

```diff
@@ -27,8 +28,15 @@ const Styled = {
 };
 
 export const Header = () => {
-    const [isPersonOpen, setIsPersonOpen] = useState(false);
+    const [isPersonSheetOpen, setIsPersonSheetOpen] = useState(false);
     const [isBookmarkOpen, setIsBookmarkOpen] = useState(false);
+    const currentUser = useCurrentUser();
+
+    const azureId: string | undefined = useMemo(() => {
+        if (currentUser) {
+            return currentUser.localAccountId;
+        }
+    }, [currentUser]);
```

Comment:
```suggestion
    const azureId = currentUser?.localAccountId
```
no need to memorized

Reply:
Done